apptail/instance.go

```diff
@@ -79,10 +103,16 @@ func (instance *Instance) tailFile(name, filename string, stopCh chan bool) {
 		MaxLineSize: GetConfig().MaxRecordSize,
 		MustExist:   true,
 		Follow:      true,
-		Location:    &tail.SeekInfo{-limit, os.SEEK_END},
+		Location:    location,
 		ReOpen:      false,
 		Poll:        false,
 		RateLimiter: rateLimiter})
+
+	// IMPORTANT: this registration happens everytime app restarts
+	if shouldInitialize {
+		tracker.InitializeChildNode(instance.getShortDockerId(), filename, t.Location.Offset)
```

Comment:
> , so we need a real Tell() if the file already exists (old container with no known state).
Tell() throws exception currently if the file has not been tailed yet(which is not a bug, accepted behaviour).